### PR TITLE
Setting up the starting point for every three people

### LINKED ISSUE #1: Select an mp3 music file for the homepage
Select and setup a music file that is different than the game music and have it play on the homepage

### PR DESCRIPTION
**Pull request by power-pole03**
Have put all the files that we can start all equally

### CODE REVIEW

File: public/Catch/fonts/Truculenta-Black.ttf
Comment (ryanprice) [inline public/Catch/fonts/Truculenta-Black.ttf:1]: Can you consolidate all the font files to 1 copy across the repository ? that each game uses

File: public/Match/images/.DS_Store
Comment (ryanprice) [inline public/Match/images/.DS_Store:1]: remove pls

File: public/Match/musics/track1.wav
Comment (ryanprice) [inline public/Match/musics/track1.wav:1]: remove all music tracks you dont use

File: public/images/1_or_11.png
Comment (ryanprice) [inline public/images/1_or_11.png:1]: Use the same images for all 3 games, in 1 general folder

**Review by ryanprice — CHANGES_REQUESTED** [review]
review all

File: public/Match/images/.DS_Store
Comment (power-pole03) [reply public/Match/images/.DS_Store:1]: Removed .DS_Store file from images folder

File: public/Catch/fonts/Truculenta-Black.ttf
Comment (yuyoungKim) [reply public/Catch/fonts/Truculenta-Black.ttf:1]: I consolidate all the fonts, sounds, and images.

Comment (yuyoungKim) [reply public/Catch/fonts/Truculenta-Black.ttf:1]: Also, we chose one mp3 music file for the homepage as well.

File: public/Match/musics/track1.wav
Comment (yuyoungKim) [reply public/Match/musics/track1.wav:1]: We removed all tracks

File: public/images/1_or_11.png
Comment (yuyoungKim) [reply public/images/1_or_11.png:1]: Yes, we merged them into 1 general folder

File: .DS_Store
Comment (ryanprice) [inline .DS_Store:1]: remove pls

**Review by ryanprice — APPROVED** [review]
looks good

**Pull request merged**